be/src/vec/exec/join/vhash_join_node.cpp

```diff
@@ -601,6 +601,9 @@ HashJoinNode::HashJoinNode(ObjectPool* pool, const TPlanNode& tnode, const Descr
           _is_outer_join(_match_all_build || _match_all_probe) {
     _runtime_filter_descs = tnode.runtime_filters;
     init_join_op();
+
+    // avoid vector expand change block address.
+    _build_blocks.reserve(128);
```

Comment:
How about if size of `_build_blocks` bigger than 128?

Reply:
> How about if size of `_build_blocks` bigger than 128?

one block can store 4g data, i think 128*4g is big enough to execute one query. btw, if probe data bigger than 512g, runtime filter maybe will core dump when insert data.